include/libkrun.h

```diff
@@ -582,10 +582,16 @@ int32_t krun_split_irqchip(uint32_t ctx_id, bool enable);
  * Arguments:
  *  "ctx_id" - the configuration context ID.
  *
- * Returns:
+ * Notes:
  *  This function only returns if an error happens before starting the microVM. Otherwise, the
  *  VMM assumes it has full control of the process, and will call to exit() once the microVM shuts
  *  down.
+ *
+ * Returns:
+ *  -EINVAL - The VMM has detected an error in the microVM configuration.
+ *  125     - "init" cannot set up the environment inside the microVM.
```

Comment:
Should we define these as enums available in `libkrun.h`?

Reply:
Both in podman and chroot they're just referred directly by their number, so I think us giving them a name would generate confusion more than anything else.

Reply:
Ok sounds good to me.